src/java/org/apache/cassandra/index/sai/disk/v1/V1SSTableIndex.java

```diff
@@ -109,7 +109,7 @@ public V1SSTableIndex(SSTableContext sstableContext, StorageAttachedIndex index)
         catch (Throwable t)
         {
             FileUtils.closeQuietly(indexFiles);
-            FileUtils.closeQuietly(sstableContext);
+            FileUtils.closeQuietly(super.sstableContext);
```

Comment:
I'll probably have to port this to 5.0 as well...but for now it's stabilizing tests for this patch...